rest/route/task.go

```diff
@@ -297,3 +297,69 @@ func (tep *taskExecutionPatchHandler) Run(ctx context.Context) gimlet.Responder
 
 	return gimlet.NewJSONResponse(taskModel)
 }
+
+// GET /tasks/{task_id}/s3_path
```

Comment:
can we take "s3" out of the public API names?

Reply:
Done. I did some renaming to name things as the "task sync" feature.